Student_BCOs/June_Beshea_BCO_000550.json

```diff
@@ -0,0 +1,249 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000550",
+    "checksum": "d40f08459ef0185dbab37ca4f1221be5c9899ff79b13598c62a7e335aeedf3a7",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Correlating miRNA and gene expression data in 34 cancer phenotypes",
+        "version": "1.0.0",
+        "license": "Creative Commons Attribution 4.0 International License",
+        "created": "2019-10-17T02:13:35.166914",
+        "modified": "2019-10-17T14:42:02.817736",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "2019-10-16"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "June Beshea",
+                "orcid": "https://orcid.org/0000-0001-9101-8000",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "jbeshea@gwu.edu"
+            },
+            {
+                "name": "Ling Sheng Loh ",
+                "orcid": "https://orcid.org/0000-0003-0981-7984",
+                "affiliation": "George Washington University ",
+                "contribution": [
+                    "curatedBy"
+                ],
+                "email": "lingsloh@gwmail.gwu.edu"
+            },
+            {
+                "name": "Micah Madrid",
+                "orcid": "https://orcid.org/0000-0002-7527-022X",
+                "affiliation": "George Washington University ",
+                "contribution": [],
+                "email": "mmadrid@gwu.edu"
```

Comment:
When I reviewed your first pull request (#189 ), I commented 
![image](https://user-images.githubusercontent.com/23385872/67045119-a6f45a00-f0fb-11e9-8008-240eb5b6fe5d.png)

Why have
1) your team members not been removed from contributors
2) The publication authors have not been added.